docs/SUMMARY.md

```diff
@@ -108,13 +109,15 @@
 * [Postprocessing variant calls](postprocessing_variant_calls/README.md)
   * [vardict_filter_case-control 0.1.3](postprocessing_variant_calls/vardict_filter_case-control_0.1.3.md)
   * [vardict_filter_single-sample 0.1.3](postprocessing_variant_calls/vardict_filter_single-sample_0.1.3.md)
-
+  * [maf_annotated_by_bed](postprocessing_variant_calls/maf_annotated_by_bed.md)
```

Comment:
We should specify the version and please move the file replacing `postprocessing_variant_calls/0.1.7/maf_annotated_by_bed/ReadMe.md` and thus this path will become `postprocessing_variant_calls/0.1.7/maf_annotated_by_bed/maf_annotated_by_bed.md` as we don't need two documentation files